src/types/controllers/PaymentsActivationRequest.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * PaymentsActivationRequest
+ * Define the request received from CD App to require a payment activation
+ */
+
+import * as t from "io-ts";
+
+export const PaymentsActivationRequest = t.interface({});
```

Comment:
what's the purpose of these types ? t.interface({}); == t.object()

if they're stubs (to be filled out later) please add a comment : )